packages/babel-core/src/config/validation/options.js

```diff
@@ -334,11 +334,12 @@ export type NestingPath = RootPath | OverridesPath | EnvPath;
 export const assumptionsNames = new Set<string>([
   "ignoreFunctionLength",
   "ignoreToPrimitiveHint",
+  "noDocumentAll",
   "mutableTemplateObject",
```

Comment:
nit: hijklmn ... 🤪

Reply:
Oh I even moved this to "keep it in order" 😂